### DISCUSSION TITLE
AR button not showing up on ANdroid

### DISCUSSION THREAD

**Discussion by JacobInCode** [google/model-viewer#1852] [Q&A]
Hi I'm running into an issue where users aren't seeing the AR button for .glb on ANdroid - any ideas?

code looks like this 

```
          <div className="single" style={{ display: "flex", height: "100%", width: '100%'}} >
            <model-viewer
              style={{
                height: "100%",
                width: '100%'
              }}
              id={nft.collection[0].id}
              alt={nft.collection[0].alt}
              src={proxyServer + nft.collection[0].glb}
              auto-rotate
              camera-controls
              ar
              ar-modes="quick-look"
              ar-scale="auto"
              ios-src={nft.collection[0].id}
            >
              <button slot="ar-button" className="ar-button" /> */}
            </model-viewer>
          </div>
```

**Comment by elalish** [comment]
Yes, it's because you specified `ar-modes="quick-look"`, and QuickLook only exists on iOS. If you simply omit that attribute it should work.

**Comment by JacobInCode** [comment]
will it still work in ios as well when there is a usdz if i omit that? or should i do - `  ar-modes="webxr scene-viewer quick-look" 

`

**Reply by elalish** [reply]
Yep, note the default includes everything: https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-attributes-arModes
